src/data/Dhis2TrackedEntityInstances.ts

```diff
@@ -339,10 +339,10 @@ async function getApiEvents(
 
             return {
                 event: data.id,
-                trackedEntityInstance: teiId,
```

Comment:
I think the error is here. We're using the new api for getting the events, but the import is still using the old endpoint (see `postEvents` method at `src/data/Dhis2Events.ts`).

![image](https://github.com/EyeSeeTea/Bulk-Load/assets/461124/24a9e7db-17dd-4328-8d6b-d51cc995e5e2)

![image](https://github.com/EyeSeeTea/Bulk-Load/assets/461124/f78163a7-38cc-437e-a731-4e485f77e463)

```json
"importSummaries": [
    {
        "responseType": "ImportSummary",
        "status": "ERROR",
        "description": "Event.trackedEntityInstance does not point to a valid tracked entity instance: null",
        "importCount": {
            "imported": 0,
            "updated": 0,
            "ignored": 1,
            "deleted": 0
        },
        "conflicts": [],
        "reference": "zGEzCeey4uE"
    }
],
```

For the old endpoint we need to send `trackedEntityInstance` instead `trackedEntity`.